packages/loom-plugin-build-library-extended/src/tests/plugin-build-library-extended.graphql.test.ts

```diff
@@ -0,0 +1,113 @@
+import {resolve} from 'path';
+
+import {
+  withWorkspace,
+  generateUniqueWorkspaceID,
+} from '../../../../tests/utilities';
+
+const configContent = ({graphql = false}: {graphql?: boolean} = {}) => {
+  return `
+import {createPackage, Runtime} from '@shopify/loom';
+import {
+  buildLibrary,
+  buildLibraryWorkspace,
+} from '@shopify/loom-plugin-build-library';
+import {
+  buildLibraryExtended,
+  buildLibraryExtendedWorkspace,
+} from '@shopify/loom-plugin-build-library-extended';
+export default createPackage((pkg) => {
+  pkg.entry({root: './src/index'});
```

Comment:
Once you rebase atop main this will need to become: 

```suggestion
  pkg.entry({root: './src/index.js'});
```

Reply:
Went to rebase off of main and nothing is new so this is valid as of right now.

Reply:
Ah right, the change for that is in #262